app/views/shared/_flashes.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if notice %>
+<div class="flash-container" id="dismiss-notice">
+  <div class="flash">
+    <p class="flash__message"><%= notice %></p>
+    <a href="#dismiss-notice" class="flash__dismiss"><span class="icon icon-close"></span></a>
+  </div>
+</div>
+<% end %>
+
+<% if alert %>
+<div class="flash-container" id="dismiss-notice">
+  <div class="flash flash--error">
+    <p class="flash__message"><%= alert %></p>
+    <a href="#dismiss-notice" class="flash__dismiss"><span class="icon icon-close"></span></a>
+  </div>
+</div>
+<% end %>
```

Comment:
I just copied this from GCF, which uses ERB. I think this flash stuff should be refactored anyway, so I was hesitant to put time into chaging over to HAML. Created a card for this here:  https://trello.com/c/FPMNPuGm